src/express/commands/cleanup.js

```diff
@@ -28,16 +28,17 @@ export async function stopServices(doc) {
   const stopServiceTasks = nodeIps.map(async (ip) => {
     if (isHostMap.get(ip)) {
       console.log('📍Stopping ganache on machine ' + ip + ' ...')
-      const command = 'sudo systemctl stop ganache.service'
+      const command =
+        'sudo systemctl stop ganache.service || echo "ganache not running on current machine..."'
       await runSshCommand(ip, command, maxRetries)
     }
 
-    console.log('📍Stopping heimdall on machine ' + ip + '...')
-    let command = 'sudo systemctl stop heimdalld.service'
+    console.log('📍Stopping bor on machine ' + ip + ' ...')
+    let command = 'sudo systemctl stop bor.service'
     await runSshCommand(ip, command, maxRetries)
 
-    console.log('📍Stopping bor on machine ' + ip + ' ...')
-    command = 'sudo systemctl stop bor.service'
+    console.log('📍Stopping heimdall on machine ' + ip + '...')
+    command = 'sudo systemctl stop heimdalld.service'
     await runSshCommand(ip, command, maxRetries)
```

Comment:
fine with me but curious why we changed that to stop `bor` first and then `heimdall`

Reply:
Well , I observed that when heimdall is shut down first, bor keeps trying to fetch span details even if asked to stop and the SIGTERM times out leading to a forceful killing of the process and not a graceful shutdown. I think we might need to address this on the client stability level, but for now this should be good.

Reply:
Nice